bcgov/mds
# MDS-2884: Batch Downloading of NoW Docs

# Main

- Ticket: https://bcmines.atlassian.net/jira/software/c/projects/MDS/boards/1?modal=detail&selectedIssue=MDS-2884
- Add new "Download Documents" button on Manage Documents tab, right of the tab header
- Create new modal based off of the Referral/Consultation package download package modal
- Fixed progress bar to update correctly as files get downloaded

# Other

- Disregard docgen commits in this ticket, those will be resolved before this gets merged in

# How to test

- Visit a NoW that has the "Manage Documents" tab available
- Click on the "Download Documents" button to the right of the tab header
![Core-manage-documents](https://user-images.githubusercontent.com/13891487/156076788-2f8bcb4f-2a49-4cf2-98ca-d426fc4c50fb.png)
- Verify all documents from all sections of that tab show up in the modal
![download-now-documents](https://user-images.githubusercontent.com/13891487/156076872-efb15292-62bb-4bfa-940a-22b33394edd6.png)
- Click "Download Selected Documents" and ensure the progress widget updates accordingly and that the success toast upon completion reflects the correct number of files downloaded
